### PR TITLE
Log ADC values

### PR DESCRIPTION
**Pull request by itslinotlie**
Very crude solution but it gets the job done. Adcc and Fvr are mcc_generated_files, but the parts I can explain:
- the adc source clock is fosc with a conversion clock (?) at FOSC/100
  - for the ```on-the-pad-charging``` board, this value had to change to allow adc to provide an accurate reading
- the sensor_id values need to change but I don't know what we want them to be (BATT_VOLT = ?, AUX_CUR = BUS_CURR?)

Testing:
Turned off external power, sensor_mag_2 (aux_cur) turned off but the battery was still powering everything so parsley time message continued
image

sensor_mag_1 (batt_volt) started at 4095 mV and by the time the testing finished (~5min), it dropped to 3756 mV, not sure if this is normal behaviour or requires further testing?

### CODE REVIEW

File: grandpapa_board.X/main.c [hunk 1]
```diff
@@ -193,6 +201,23 @@ static void send_status_ok_mcp(void) {
     mcp_can_send(&board_stat_msg);
 }
 
+static void send_log_msgs(void) {
+    can_msg_t batt_cur_msg;
+    uint16_t batt_cur = ADCC_GetSingleConversion(channel_BATT_CURR)/2;
```
Comment (Joe-Joe-Joe-Joe) [inline grandpapa_board.X/main.c:206]: Magic numbers should not be living in the code like this. When I read this code, I know that you're dividing by 2 based on component values on the board, but anyone else probably doesn't. Consider putting this into a macro (`#define CURR_AMP_FACTOR 0.5`) and perhaps a comment indicating where the value comes from

File: grandpapa_board.X/spi_shit.c [hunk 2]
```diff
@@ -40,4 +40,12 @@ void spi_init(){
      RC2PPS = 0b011111; //Set pin C2 as SDO (MOSI)
      RC3PPS = 0b011110; //Set pin C3 as SCK
      SPI1SDIPPS = 0b10100; //Set pin C4 as SDI (MISO)
+
+    //  these don't belong here but i need to define somewhere
```
Comment (Joe-Joe-Joe-Joe) [inline grandpapa_board.X/spi_shit.c:44]: I agree, these do not belong here

**Review by Joe-Joe-Joe-Joe — CHANGES_REQUESTED** [review]
In general, commit names should be descriptive (yes, I am sometimes guilty of the same thing). Saying something like "Read current amps from ADC" tells me exactly what functionality has been added when I go digging through the commit history for whatever reason.

Regardless, I'm very pleased and I appreciate your helping out.

**Comment by itslinotlie** [comment]
Closed because someone else is going to be implementing these changes

**Pull request closed without merging**